test/Parse/raw_string.swift

```diff
@@ -0,0 +1,128 @@
+// RUN: %target-swift-frontend -dump-ast %s 2>&1 | %FileCheck %s
+
+import Swift
+
+_ = #"""
+###################################################################
+## This source file is part of the Swift.org open source project ##
+###################################################################
+"""#
+
+// CHECK: "###################################################################\n## This source file is part of the Swift.org open source project ##\n###################################################################"
```

Comment:
You might be able to use regular expressions in the `CHECK:` directive.

```swift
// CHECK: "{{#{67}}}\n## This source file is part of the Swift.org open source project ##\n{{#{67}}}"
```

<https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-pattern-matching-syntax>

Or shorter raw strings which test the same idea. For example, using Markdown headers.

```swift
_ = #"""
    # H1 #
    ## H2 ##
    ### H3 ###
    """#
// CHECK: "# H1 #\n## H2 ##\n### H3 ###"
```

Do any of the tests require the `--strict-whitespace` option?

<https://llvm.org/docs/CommandGuide/FileCheck.html#cmdoption-strict-whitespace>

Reply:
Couldn’t get the regex to work but I added your extra test and --strict-whitespace. Thanks